corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -6,7 +6,6 @@
 requires imports/proptable.html to be included in the main template
 {% endcomment %}
 {% include "imports/fancy-code.html" %}
-{% include 'cloudcare/includes/touchforms-inline.html' %}
```

Comment:
all of this is pretty much undoing https://github.com/dimagi/commcare-hq/pull/4567